### ISSUE TITLE
Problem: popper.js in bundle and downloaded as script in theme.html (20kb)

### ISSUE THREAD

**Issue by janat08** [Blockrazor/blockrazor#1439]
Problem: Popper.js is still in bundle, even if theme.html is supposed to be dynamically imported. The downloaded version (self-served) is in ../ui/components/compatibility which is dynamically imported. I don't know if script tags count towards the bundle.

**Comment by gazhayes** [comment]
theme.html can be removed, it's just to test bootswatch.

**Comment by anbud** [comment]
@blockrazorbot claim

**Comment by anbud** [comment]
I've just tried this one, `popper.js` is actually imported by `bootstrap` npm package. In order to fix this one, I've added `bootstrap` to dynamically imported packages in #1460. It doesn't seem to cause any issues with the UI, I've tried it in multiple browsers to ensure that I didn't have anything cached. If it does cause problems in the production, we can easily revert it. If not, we gain more performance.
